terraform/prod/main.tf

```diff
@@ -283,6 +283,11 @@ module "integrated_data_avl_data_producer_api" {
   internal_data_endpoint      = local.secrets["internal_avl_ingestion_nlb_ip"]
 }
 
+module "integrated_data_avl_data_consumer_api" {
```

Comment:
And we could maybe move creating an instance of the module into the `avl-pipeline` so we can easily pass the outputs through to the `siri-vm-api` module